lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -190,6 +190,12 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
      * */
     downloadPromptReminderDelay: number = 15;
 
+    /**
+     * Emitted when user clicks on download button on non responsive preview dialog.
+     * */
+    @Output()
+    downloadFile: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
if you add an event you should add it also in the readme

Reply:
Added downloadFile Event to the readme for viewer.component